src/main/java/org/repodriller/scm/SubversionRepository.java

```diff
@@ -169,10 +172,14 @@ public SCMRepository info() {
 		}
 	}
 
+	public Commit getCommit(String id) {
+		return this.getCommit(id, Arrays.asList(new NoDiffFilter()));
+	}
+	
 	@SuppressWarnings("rawtypes")
 	@Override
 	/* TODO Refactor as in GitRepository.getCommit. */
-	public Commit getCommit(String id) {
+	public Commit getCommit(String id, List<DiffFilter> diffFilters) {
```

Comment:
at least throw a RuntimeException to let the user know that the feature is not implemented, when `!diffFilters.isEmpty()`

Reply:
Would it be better to check if the list contains any filters that are **not** `NoDiffFilter`? 

The method `getCommit(String)` calls this method with `NoDiffFilter` as the only filter. I did that since this is a public method, and that makes this not be a breaking change.

Reply:
> Would it be better to check if the list contains any filters that are not NoDiffFilter?

Sounds hacky.

> The method `getCommit(String)` calls this method with `NoDiffFilter` as the only filter.

Why doesn't it call it with an empty list? Then throw an exception if the list is not empty.

Reply:
> The method getCommit(String) calls this method with NoDiffFilter as the only filter. 

You're right. I got distracted because `GitRepository` does that ⬆️